index.html

```diff
@@ -335,49 +365,27 @@ <h2 id="coordination"> Dependencies and Liaisons </h2>
-          worked upon by the Device and Sensors Working Group. This Working
-          Group will liaise with the Device and Sensors Working Group, in
-          particular to help ensure consistency of supported service discovery
-          mechanisms when a user agent implements both APIs. </p>
         <p> No external dependencies against the specifications of this Working
           Group have been identified. </p>
         <h3 id="liaisons"> Liaisons </h3>
-        <p> The Working Group expects to maintain contacts with at least the
-          following groups and Activities within W3C (in alphabetical order) and
-          ask for wide reviews of its deliverables before publication as
-          Candidate Recommendation, and where appropriate: </p>
+        <p>For all specifications, this Working Group will seek <a href="https://www.w3.org/Guide/Charter.html#horizontal-review">horizontal review</a> for accessibility, internationalization, performance, privacy, and security with the relevant Working and Interest Groups, and with the <a title="Technical Architecture Group" href="https://www.w3.org/2001/tag/">TAG</a>. Invitation for review must be issued during each major standards-track document transition, including <a title="First Public Working Draft" href="https://www.w3.org/2017/Process-20170301/#RecsWD">FPWD</a> and at least 3 months before <a title="Candidate Recommendation" href="https://www.w3.org/2017/Process-20170301/#RecsCR">CR</a>, and should be issued when major changes occur in a specification.</p>
+        <p>Additional technical coordination with the following Groups will be made, per the <a href="https://www.w3.org/2017/Process-20170301/#WGCharter">W3C Process Document</a>:</p>
         <dl>
-          <dt><a href="http://www.w3.org/2009/dap/" id="dap"> Device and Sensors
-              Working Group </a></dt>
-          <dd> The Device and Sensors Working Group defines the Network Service
-            Discovery API that addresses some of the use cases that are in scope
-            of the Second Screen Working Group. </dd>
-          <dt><a href="http://www.w3.org/html/wg/" id="hme"> HTML Media
-              Extensions Working Group </a></dt>
-          <dd> The Second Screen Working Group may interact with the HTML Media
-            Extensions Working Group on the interaction between media extensions
-            and the Remote Playback API specification. </dd>
-          <dt><a href="http://www.w3.org/Privacy/"> Privacy Interest Group </a></dt>
-          <dd> The Second Screen Working Group intends to secure reviews on its
-            deliverables from the Privacy Interest Group to help ensure they
-            offer the right level of protection to users. </dd>
           <dt><a href="http://www.w3.org/community/webscreens/" id="sspcg">
               Second Screen Community Group </a></dt>
           <dd> This group developed the initial version of the Presentation API
```

Comment:
/The Community Group will specify a set of network protocols that may warrant changes in the Presentation API./The Community Group incubates the Open Screen Protocol for the APIs defined in this Working Group. This work on protocols may warrant changes in the Presentation API.